style.css

```diff
@@ -264,42 +265,16 @@ header a {
   line-height: 20px;
 }
 
-.see-project-btn {
+#multipost-btn {
   width: 124px;
   height: 48px;
-  text-align: center;
-  font-family: "Inter", serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  letter-spacing: 0.03em;
-  color: #fff;
-  background: #ff6b00;
-  border: none;
-  transition: all 5s ease 0.25s;
 }
 
-.grid-container {
+.grid-container1 {
   display: flex;
```

Comment:
- [ ] Great job with the CSS. We have to try our best to follow [DRY principles](https://vanseodesign.com/css/dry-principles/). I see display: flex appearing so many times in your style.css. You can improve this by add flex as a class in your html and creating css class for that. Let me illustrate below:

/index.html [Line 139]
` <div class="grid-container1 flex">`

The do a css class of flex
```
.flex {
      display: flex
       }
```